go/vt/vttablet/tabletserver/query_executor.go

```diff
@@ -97,18 +97,12 @@ func (qre *QueryExecutor) Execute() (reply *sqltypes.Result, err error) {
 	}
 
 	switch qre.plan.PlanID {
-	case planbuilder.PlanDDL:
-		return qre.execDDL()
 	case planbuilder.PlanNextval:
 		return qre.execNextval()
 	case planbuilder.PlanSelectImpossible:
 		if qre.plan.Fields != nil {
 			return &sqltypes.Result{
-				Fields:       qre.plan.Fields,
-				RowsAffected: 0,
-				InsertID:     0,
-				Rows:         nil,
-				Extras:       nil,
+				Fields: qre.plan.Fields,
 			}, nil
 		}
 	}
```

Comment:
case planbuilder.PlanSelect, planbuilder.PlanSelectImpossible also exists below. Why "planbuilder.PlanSelectImpossible" is present in two switch statement?